wcfsetup/install/files/js/WoltLabSuite/Core/Ui/ItemList.js

```diff
@@ -110,7 +110,7 @@ define(['Core', 'Dictionary', 'Language', 'Dom/Traverse', 'EventKey', 'WoltLabSu
 							for (var i = 0, length = values.length; i < length; i++) {
 								input = elCreate('input');
 								input.type = 'hidden';
-								input.name = options.submitFieldName.replace(/{$objectId}/, values[i].objectId);
+								input.name = options.submitFieldName.replace(/{\$objectId}/, values[i].objectId);
```

Comment:
Will the braces characters also need to be encoded? They have a special meaning in a regular expression as well.

Reply:
Curly braces do have a special meaning as well, thats true. But they only quantify a group or equal in a regex. In this case here, the above regex matches exactly the string `{$objectId}`. But I can add an escape-character here too, so that will never be a problem. Will add it.